packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,4 +2,4 @@
 
 ## Unreleased
 
-* Initial stable release
+* Initial stable release!
```

Comment:
Its back!

Reply:
its weird with this being unreleased! Maybe I should filter this package out of the change log checker for now

Reply:
lol, I get it now. And yes, I'd vote you can filter it out.